manifests/init.pp

```diff
@@ -94,7 +94,7 @@
 
   nginx::resource::server { 'webhook':
     server_name          => [$canonical_fqdn],
-    spdy                 => 'on',
+    http2                => 'on',
```

Comment:
Do stock CentOS and Debian provide new enough versions of Nginx that we can ignore the spdy bit or does it require extra repos?

Reply:
1.9.5 was released 5 years ago and from a quick look it appears all non EoL versions of Debian and CentOS ship with a version that supports http2.